tests/app/service/test_rest.py

```diff
@@ -3385,3 +3390,122 @@ def test_get_returned_letter_summary(admin_request, sample_service):
+    letter_from_job = create_notification(template=sample_letter_template, client_reference='letter_from_job',
+                                          status=NOTIFICATION_RETURNED_LETTER,
+                                          job=job, job_row_number=2,
+                                          created_at=datetime.utcnow() - timedelta(days=1),
+                                          created_by_id=sample_letter_template.service.users[0].id)
+    create_returned_letter(service=sample_letter_template.service, reported_at=datetime.utcnow(),
+                           notification_id=letter_from_job.id)
+
+    one_off_letter = create_notification(template=sample_letter_template,
+                                         status=NOTIFICATION_RETURNED_LETTER,
+                                         created_at=datetime.utcnow() - timedelta(days=2),
+                                         created_by_id=sample_letter_template.service.users[0].id)
+    create_returned_letter(service=sample_letter_template.service, reported_at=datetime.utcnow(),
+                           notification_id=one_off_letter.id)
+
+    api_key = create_api_key(service=sample_letter_template.service)
+    api_letter = create_notification(template=sample_letter_template, client_reference='api_letter',
+                                     status=NOTIFICATION_RETURNED_LETTER,
+                                     created_at=datetime.utcnow() - timedelta(days=3),
+                                     api_key=api_key)
+    create_returned_letter(service=sample_letter_template.service, reported_at=datetime.utcnow(),
+                           notification_id=api_letter.id)
+
+    precompiled_template = create_template(service=sample_letter_template.service, template_type='letter', hidden=True,
+                                           template_name='hidden template')
+    precompiled_letter = create_notification_history(template=precompiled_template, api_key=api_key,
+                                                     client_reference='precompiled letter',
+                                                     created_at=datetime.utcnow() - timedelta(days=4))
+    create_returned_letter(service=sample_letter_template.service, reported_at=datetime.utcnow(),
+                           notification_id=precompiled_letter.id)
+
+    uploaded_letter = create_notification_history(template=precompiled_template, client_reference='filename.pdf',
+                                                  created_at=datetime.utcnow() - timedelta(days=5),
+                                                  created_by_id=sample_letter_template.service.users[0].id)
+    create_returned_letter(service=sample_letter_template.service, reported_at=datetime.utcnow(),
+                           notification_id=uploaded_letter.id)
+
+    not_included_in_results = create_template(service=create_service(service_name='not included in results'),
```

Comment:
Could we name this variable `not_included_in_results_template` please for consistency